infrastructure/kubernetes-demo/hello-node-eks/index.js

```diff
@@ -3,9 +3,27 @@ const app = express()
 const port = 3000
 
 app.get('/', (req, res) => {
-  res.send('Hello Node!')
+
+  res.send(`Hello Node!`);
+})
+
+app.get('/check', async (req, res) => {
+  const {SecretsManagerClient, GetSecretValueCommand} = require('@aws-sdk/client-secrets-manager');
```

Comment:
Would these make sense to go at the top of the file?

I guess if they're here they only appear in the `/check` endpoint? and `/` certainly doesn't need those deps... but then feels a bit odd to be requiring stuff in within a function 🤔. I went looking for these in the opening lines, perhaps that's just me.

Or perhaps in aprod app we'd move these intot wo different files?